data/SandipPalit.json

```
@@ -80,6 +86,14 @@
     }
   ],
   "milestones": [
+    {
+      "title": "Started my own Community",
+      "date": "June 2023",
+      "icon": "SiPostman",
+      "color": "blue",
+      "description": "Postman Community - Kolkata is a API focessed community in the City Of Joy.",
```

[Nitpick] I think it would be great if you mentioned why or what inspired you to start this community. It'll help people learn more about you. 
```suggestion
      "description": "Postman Community - Kolkata is a API focessed community in the City Of Joy.",
```

---

@CBID2 in future I would suggest that these comments continue to be an `inline comment` but not an `inline suggestion` given that you have not actually made any suggested changes (which in any event we do not recommend for an author's content).